### PR TITLE
Rethrowing an original error if the error message is not matched to the video text regEx

### PR DESCRIPTION
**Pull request by OleksandrPodoliako**
Added rethrowing an error if  there's no match.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Thanks for contributing. The PR needs a better title and I think you could [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) up the code a bit.

**Comment by OleksandrPodoliako** [comment]
@sindresorhus, no problem. I've made changes.

File: main/utils/encoding.js [hunk 1]
```diff
@@ -14,7 +14,12 @@ const getEncoding = async filePath => {
   try {
     await execa(ffmpegPath, ['-i', filePath]);
   } catch (error) {
-    return /.*: Video: (.*?) \(.*/.exec(error.stderr)[1];
+    const errorMessageMatchedText = /.*: Video: (.*?) \(.*/.exec(error.stderr);
+    if (errorMessageMatchedText) {
+      throw error;
+    } else {
```
Comment (sindresorhus) [inline main/utils/encoding.js:20]: The `else` here is not needed.

File: main/utils/encoding.js [hunk 2]
```diff
@@ -14,7 +14,12 @@ const getEncoding = async filePath => {
   try {
     await execa(ffmpegPath, ['-i', filePath]);
   } catch (error) {
-    return /.*: Video: (.*?) \(.*/.exec(error.stderr)[1];
+    const errorMessageMatchedText = /.*: Video: (.*?) \(.*/.exec(error.stderr);
+    if (errorMessageMatchedText) {
+      throw error;
+    }
+
+  return errorMessageMatchedText[1];
```
Comment (sindresorhus) [inline main/utils/encoding.js:22]: Indentation

File: main/utils/encoding.js [hunk 3]
```diff
@@ -14,7 +14,12 @@ const getEncoding = async filePath => {
   try {
     await execa(ffmpegPath, ['-i', filePath]);
   } catch (error) {
-    return /.*: Video: (.*?) \(.*/.exec(error.stderr)[1];
+    const errorMessageMatchedText = /.*: Video: (.*?) \(.*/.exec(error.stderr);
+    if (errorMessageMatchedText) {
```
Comment (sindresorhus) [inline main/utils/encoding.js:18]: Shouldn't this be negated?

File: main/utils/encoding.js (hunk 2 above)
Comment (OleksandrPodoliako) [reply main/utils/encoding.js]: I was wondering if you can give me a hint how to set up code formatting  automatically in VSC?

File: main/utils/encoding.js (hunk 3 above)
Comment (OleksandrPodoliako) [reply main/utils/encoding.js]: Correct

File: main/utils/encoding.js [hunk 4]
```diff
@@ -6,21 +6,26 @@ const ffmpeg = require('@ffmpeg-installer/ffmpeg');
 const util = require('electron-util');
 const execa = require('execa');
 
-const {track} = require('../common/analytics');
+const { track } = require('../common/analytics');
```
Comment (sindresorhus) [inline main/utils/encoding.js:9]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
Does this fix an issue? If so, it would be great if you could link to it. Also, make sure your changes actually fixes the issue by running the PR and manually confirming.

File: main/utils/encoding.js (hunk 4 above)
Comment (OleksandrPodoliako) [reply main/utils/encoding.js]: I got it

**Comment by OleksandrPodoliako** [comment]
I've checked, now it works properly.
